datahub-web-react/src/app/ingest/source/IngestionSourceTableColumns.tsx

```diff
@@ -0,0 +1,186 @@
+import { blue } from '@ant-design/colors';
+import { CodeOutlined, CopyOutlined, DeleteOutlined } from '@ant-design/icons';
+import { Button, Image, Tooltip, Typography } from 'antd';
+import cronstrue from 'cronstrue';
+import React from 'react';
+import styled from 'styled-components/macro';
+import { ANTD_GRAY } from '../../entity/shared/constants';
+import { capitalizeFirstLetter } from '../../shared/textUtil';
+import {
+    getExecutionRequestStatusDisplayColor,
+    getExecutionRequestStatusDisplayText,
+    getExecutionRequestStatusIcon,
+    RUNNING,
+    sourceTypeToIconUrl,
+} from './utils';
+
+const PreviewImage = styled(Image)`
+    max-height: 28px;
+    width: auto;
+    object-fit: contain;
+    margin: 0px;
+    background-color: transparent;
+`;
+
+const StatusContainer = styled.div`
+    display: flex;
+    justify-content: left;
+    align-items: center;
+`;
+
+const StatusButton = styled(Button)`
+    padding: 0px;
+    margin: 0px;
+`;
+
+const ActionButtonContainer = styled.div`
+    display: flex;
+    justify-content: right;
+`;
+
+const TypeWrapper = styled.div`
+    align-items: center;
+    display: flex;
+`;
+
+const CliBadge = styled.span`
+    margin-left: 20px;
+    border-radius: 15px;
+    border: 1px solid ${ANTD_GRAY[8]};
+    padding: 1px 4px;
+    font-size: 10px;
+
+    font-size: 8px;
+    font-weight: bold;
+    letter-spacing: 0.5px;
+    border: 1px solid ${blue[6]};
+    color: ${blue[6]};
+
+    svg {
+        display: none;
+        margin-right: 5px;
+    }
+`;
+
+export function TypeColumn(type: string, record: any) {
```

Comment:
I think we should do this for all of our tables. Really like splitting columns out like this